CHANGELOG.md

```diff
@@ -137,3 +137,4 @@ Add support for AWS Transit Gateway and expose Azure config parameters
 
 ## [1.8.0] - not yet released
 - Sync Flink Table API
+- Fix required `UpdateServiceIntegrationRequest.UserConfig` 
```

Comment:
we don't update CHANGELOG.md in this repository, it's very outdated and will be dropped soon
so please revert this changelog entry, thank you!